package.json

```diff
@@ -35,6 +29,10 @@
     "prepare": "npm run build-storybook && npm run build"
   },
   "dependencies": {
+    "d3-array": "^1.2.0",
```

Comment:
It looks like these could still be marked as peer dependencies, and pangea can add them as top-level deps. Does that work?

Reply:
yep!